lobsterpy/featurize/core.py

```diff
@@ -528,9 +725,8 @@ def _calculate_wicoxx_ein(self) -> tuple[float, float, float, float]:
             icoxx = self.icoxxlist.icohpcollection.get_icohp_by_label(
                 lab, summed_spin_channels=True
             )
-            weight = (
-                icoxx / icoxx_total
-            )  # calculate the weights based on icohp contri to total icohp of the structure
+            # calculate the weights based on icohp contri to total icohp of the structure
+            weight = (icoxx / icoxx_total) if icoxx_total != 0 else 0
```

Comment:
I don't think this is safe. Please make sure you really cutoff numerical issues. Please compare the floats correctly. We had this issue multiple times before.

Reply:
Fixed it using explicit try-except block